css-layout-classes/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header>
+            <h1>Choose your character!</h1>
+          </header>
```

Comment:
You don't need a `header` element here. Normally it would be used for a page's header whereas this would just be a section on our page.